woocommerce/woocommerce-android
# [JITM] Refactor jitms so it can be used as plug & play component

Closes: #8655


### Description

The PR extracts JITM into a separate fragment so we can put this fragment into a place where we want to show the JITM without any extra actions. Also, this will be beneficial if/when we will use modal JITMs eventually

### Testing instructions

Run regression testing of JITM. It should look exactly the same as now. Also, notice that it should be prefetched when pull to refresh is done
